@uportal/dashboard-carousel/src/components/PortletRenderer.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div v-html="portletContent" />
+</template>
+
+<script>
+import ky from 'ky';
+
+export default {
+  name: 'PortletRenderer',
+  props: {
+    portletHtmlUrl: String,
+  },
+  asyncComputed: {
+    portletContent: {
+      async get() {
+        const {portletHtmlUrl, debug} = this;
+        try {
+          const headers = true
```

Comment:
`true` should be `debug` and `debug` should be included in props (lines 10-12).

Reply:
Fixed

Reply:
:confused: it's still showing as `true`